.nvmrc

```diff
@@ -0,0 +1 @@
+14
```

Comment:
when we beginn to use nvm here then we have to use yarn as npm package and not as global binary anymore